lam4doc/entrypoints/api/handlers.py

```diff
@@ -15,28 +15,34 @@
 import requests
 from flask import send_file
 from werkzeug.datastructures import FileStorage
-from werkzeug.exceptions import InternalServerError
+from werkzeug.exceptions import InternalServerError, UnprocessableEntity, BadRequest
 
 from lam4doc.adapters.sparql_adapter import FusekiSPARQLAdapter
-from lam4doc.config import LAM_DOCUMENT_PROPERTY_GRAPH, LAM_CLASSES_GRAPH, LAM_CELEX_CLASSES_GRAPH, \
-    LAM_FUSEKI_PORT, LAM_FUSEKI_LOCATION
-from lam4doc.config import LAM_LOGGER
+from lam4doc.config import LAMConfig, DEFAULT_REPORT_TYPE, REPORT_EXTENSIONS
 from lam4doc.services.handlers import generate_lam_report as service_generate_lam_report, \
     generate_indexes as service_generate_indexes, zip_files
 
-logger = logging.getLogger(LAM_LOGGER)
+config = LAMConfig()
```

Comment:
as mentioned, this statement can be moved into the config module, and then you end up simply importing it.